paperslicer/metadata/providers/pubmed.py

```
@@ -0,0 +1,119 @@
+            r = requests.get(self.efetch, params=params, timeout=self.timeout)
+            if not r.ok:
+                return []
+            parser = etree.XMLParser(recover=True)
+            root = etree.fromstring(r.content, parser)
+            kws: List[str] = []
+            for kw in root.findall(".//KeywordList/Keyword"):
+                if kw.text:
+                    txt = " ".join(kw.text.split()).strip()
+                    if txt:
+                        kws.append(txt)
+            return kws
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)